### PR TITLE
Recreate coredns deployment for adoption

### PR DESCRIPTION
**Pull request by AverageMarcus**

### CODE REVIEW

**Review by calvix — APPROVED** [review]
lgtmiwfy

**Review by whites11 — APPROVED** [review]
if it works for you

File: helm/cluster-gcp/templates/_coredns.tpl [hunk 1]
```diff
@@ -196,7 +196,7 @@ data:
       resourceNames: ["coredns"]
       verbs: ["delete"]
     - apiGroups: [""]
-      resources: ["services"]
+      resources: ["services", "deployments"]
```
Comment (fiunchinho) [inline helm/cluster-gcp/templates/_coredns.tpl:199]: `deployments` belong to the `apps` apigroup

**Review by fiunchinho — APPROVED** [review]
it works!

**Pull request merged**